mappings/services.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
```

Comment:
I'm guessing we'll still have three nodes in out cluster, each with it's own copy of the shard? I'm just thinking about if we lose a node can we recover - same as game day.